### DISCUSSION TITLE
Docker - Nas Synology

### DISCUSSION THREAD

**Discussion by SilverResort** [microrealestate/microrealestate#185] [Q&A]
Hello,

I am very interested in your project.

Is there a simple way to install it on a synology nas (docker-compose.yml file) ?
I cannot find it on the docker registry, will it be availlable on it ?

Kind regards

**Comment by camelaissani** [comment]
Hello @SilverResort ,
Nothing has be done to install this project on a synology nas so far. 
Any contributions to support it are welcome.
Thanks

**Reply by michaelsmolka** [reply]
I successfully installed it on my NAS.
Simply create the folders (i. e. volum1/docker/mre/backup , data , data/mongodb, data/redis
Create the env and docker-compose files by copying the contents from the corresponding files here in the github repo and create a new project in container manager.
I just changed the tags in the compose for mongo and redis to :latest as I got authentication errors for the given tags.
That's it.
After that you can register an account by pointing your browser to http://<yourNASIP>:8080/landlord/
